entrypoint.sh

```diff
@@ -25,10 +25,11 @@ sanitize "${IMAGE}" "image"
 sanitize "${TAG}" "tag"
 
 if [ "$REGISTRY" == "docker.pkg.github.com" ]; then
-    export IMAGE="$GITHUB_REPOSITORY/$IMAGE"
+    GITHUB_IMAGE_PREFIX="$(echo $GITHUB_REPOSITORY | tr '[:upper:]' '[:lower:]')"
```

Comment:
I'd personally rather name this variable `IMAGE_NAMESPACE`, as I think it makes more clear what it contains

```suggestion
    IMAGE_NAMESPACE="$(echo $GITHUB_REPOSITORY | tr '[:upper:]' '[:lower:]')"
```